Packs/CommonPlaybooks/Playbooks/playbook-Search_And_Delete_Emails_-_Generic_-_v2_6_1.yml

```diff
@@ -1295,9 +1373,9 @@ inputs:
   value:
     simple: "true"
   required: false
-  description: Whether to limit the search to the current week.
+  description: Whether to limit the search to the current week. Note that if you are using the LastModifiedTime filter in the O365KQL input, this input will not be taken into consideration.
```

Comment:
this is the generic playbook - so I think that better to keep this note only for the `O365KQL` input description.